doc/tutorials/cne/cne.txt

```diff
@@ -0,0 +1,356 @@
+/*!
+
+@file cne.txt
+@author Kartik Nighania
+@brief Tutorial on how to use the CNE optimizer class.
+
+@page cnetutorial CNE Optimizer tutorial (mlpack_cne)
```

Comment:
I don't think there is any `mlpack_cne` program, so maybe you should reference the CNE class here instead.